Makefile

```diff
@@ -25,7 +25,7 @@ lint:
 	go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run -c .golangci.yml
 
 docker:
-	docker build --build-arg=GIT_VERSION=$(GIT_VERSION) -t $(IMAGE_PREFIX)/unused -f Dockerfile.exporter .
+	docker build --build-arg=GIT_VERSION=$(GIT_VERSION) -t $(IMAGE_PREFIX)/unused -f Dockerfile.exporter . --load
```

Comment:
Nit: unrelated to the scope of the PR (as per the title) 😉 

No need to change it, tho.

Reply:
YES NOT DOING ANOTHER PR JUST TO SATISFY YOUR #OCD ELF
(sorry capslock stuck)

Reply:
<img src="https://media2.giphy.com/media/kIQw3431MmAuiyvqR3/giphy.gif"/>